README.md

```diff
@@ -1,2 +1,15 @@
 # conmon
+
 An OCI container runtime monitor.
+
+Conmon is a monitoring program and communication tool between a container manager (like [podman](https://podman.io/) or [CRI-O](https://cri-o.io/)) and an OCI runtime (like [runc](https://github.com/opencontainers/runc) or [crun](https://github.com/giuseppe/crun)) for a single container.
+
+Upon being launched, it double-forks to daemonize and detach from the parent that launched it. It then launches the runtime as its child.  This allows managing processes to die in the foreground, but still be able to watch over and connect to the child process (the container).
+
+While the container runs, conmon does two things:
+*Provides a socket for attaching to the container, holding open the container's standard streams and forwarding them over the socket.
```

Comment:
Probably want spaces after the asterisks so the markdown formats properly

Reply:
Yes @haircommander make sure this displays correctly.